SpatialGDK/Source/SpatialGDK/Private/SpatialView/SubView.cpp

```diff
@@ -69,6 +69,14 @@ TSet<Worker_EntityId_Key> FSubView::GetCompleteEntities() const
 	return CompleteEntitiesSet.Difference(IncompleteEntitiesSet);
 }
 
+void FSubView::Refresh()
```

Comment:
Why refresh everything in a subview instead of refreshing the entity for all subviews (as most things do)?
I'm not sure what we previously had was better although it seems like we probably want to change both of these to have the pub sub thing that's been mentioned a few times.

Reply:
The consideration here is that we don't know every entity that might be waiting on ownership completeness.

Reply:
Ok, in which case I think we add this to the evidence that we need to change how subviews work.